DESCRIPTION

```diff
@@ -24,13 +25,14 @@ Imports:
     scales,
     survival,
     stats,
+    broom,
     tidyr
 Suggests:
     knitr,
     RTCGA.clinical(>= 20151101.4.0),
     survMisc
 VignetteBuilder: knitr
-biocViews:
+biocViews: 
```

Comment:
@kassambara @pbiecek this section `biocViews`  is empty and is only used at Bioconductor. I suggest this should be removed